src/Algorithms/LGG09/reach_inhomog.jl

```diff
@@ -186,6 +198,21 @@ function reach_inhomog_LGG09!(F::Vector{RT},
     return ρmat
 end
 
+function _reach_inhomog_LGG09_invariant!(Ω₀, U, rᵢ, sᵢ, ndirs, threaded::Val{false})
+    for j in 1:ndirs
+        d = view(rᵢ, :, j)
+        ρmat[j, k] = ρ(d, Ω₀) + sᵢ[j]
+        sᵢ[j] += ρ(d, U)
+    end
+end
+
+function _reach_inhomog_LGG09_invariant!(Ω₀, U, rᵢ, sᵢ, ndirs, threaded::Val{true})
+    Threads.@threads for j in 1:ndirs
+        d = view(rᵢ, :, j)
+        ρmat[j, k] = ρ(d, Ω₀) + sᵢ[j]
```

Comment:
ρmat is undefined https://github.com/JuliaReach/ReachabilityAnalysis.jl/pull/361/checks?check_run_id=1396547261#step:5:93